.eslintignore

```diff
@@ -7,3 +7,5 @@
 **/.git/**
 **/public/**
 **/microsite/**
+**/templates/**
+**/sample-templates/**
```

Comment:
oh wait btw, could these be `**/sample-templates/**.hbs.*` instead?

Reply:
Nah, plenty of other lint issues in the other files too. The `.hbs` ones aren't actually linted by the pre-commit hook either